packages/pds/src/db/migrations/20230316T142245535Z-post-entities-to-facets.ts

```diff
@@ -0,0 +1,9 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema.alterTable('post_entity').renameTo('post_facet').execute()
```

Comment:
I remember this table is missing a primary key. Would be a good time to get one on it

But also: I don't think we actually use this table for anything. Maybe we could drop??

Reply:
Yeah good call, let's just drop this bad boy.